internal/tools/kops/run.go

```diff
@@ -40,12 +40,12 @@ func outputLogger(line string, logger log.FieldLogger) {
 	matches := glogRe.FindStringSubmatch(line)
 	if matches == nil {
 		// Assume non-glog output is a warning.
-		logger.Warnf("kops: %s", line)
+		logger.Warnf("[kops] %s", line)
```

Comment:
👍

Reply:
Glad you like that change. I find this make the logs a lot easier to understand at a glance.